guide/09-mapping-and-visualization/managing-offline-map-areas.ipynb

```diff
@@ -19,7 +19,7 @@
    "source": [
     "## Creating offline map areas\n",
     "\n",
-    "With ArcGIS API for Python, you can conveniently manage offline areas from the `WebMap` object. The `offline_areas` property off the `WebMap` object gives you access to the `OfflineAreaManager` object with which you can `create()`, `list()` and `update()` these offline packages."
+    "With ArcGIS API for Python, you can conveniently manage offline areas from the `WebMap` object. The [`offline_areas`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.mapping.html#arcgis.mapping.WebMap.offline_areas) property off the `WebMap` object gives you access to the [`OfflineAreaManager`](https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.mapping.html#offlinemapareamanager) object with which you can `create()`, `list()` and `update()` these offline packages."
```

Comment:
Very small grammar change, please change

`Using [ArcGIS Runtime SDKs](https://developers.arcgis.com/features/offline/) you can build offline...`

to

`Using [ArcGIS Runtime SDKs](https://developers.arcgis.com/features/offline/), you can build offline...`